Core/Core/InstUI/Views/Cells/TextFieldCell.swift

```diff
@@ -99,14 +98,25 @@ extension InstUI {
         }
 
         private var textField: some View {
-            TextField("", text: $text, prompt: Text(placeholder).foregroundColor(Color.placeholderGray))
-                .focused($isFocused)
-                .multilineTextAlignment(label == nil ? .leading : .trailing)
-                .font(label == nil ? .semibold16 : .regular16, lineHeight: .fit)
-                .foregroundStyle(Color.textDarkest)
-                .submitLabel(.done)
-                .accessibilityLabel(accessibilityLabel)
-                .accessibilityValue(accessibilityValue)
+            if #available(iOS 16, *) {
```

Comment:
To be updated once [iOS16 PR](https://github.com/instructure/canvas-ios/pull/2827) is merged

Reply:
updated